client/src/App.jsx

```diff
@@ -17,9 +18,9 @@ const App = () => {
   return (
     <>
       {!!stateContext.user && <Header />}
-      <Header />
       <Switch>
         <Suspense fallback={<LoadingScreen />}>
+          <Redirect to={!!stateContext.user ? '/home' : '/login'} />
```

Comment:
This doesn't look right. What if the user go to a different route within the dashboard? @SsaaaadD2 this is what you mentioned, right?

Reply:
Hmm so for now lets delete it and create different ticket for routing then ?? @aeciorc

Reply:
I think @SsaaaadD2  did adress that in his other branch

Reply:
ok so for now then should I delete this or keep it ?

Reply:
Just keep it

Reply:
Yes I changed this in the other branch